source/manual/get-started.html.md

```diff
@@ -57,10 +57,13 @@ First, install:
 * [VirtualBox][]
 * [Vagrant][]
 * The vagrant-dns plugin (`vagrant plugin install vagrant-dns`)
+* [rbenv][] (you may find it easier to [install via rbenv-installer][] than via Homebrew due to Xcode restrictions)
```

Comment:
Are the last set of square brackets needed?

Reply:
Do you mean are they needed for the markdown to be valid, or do you mean we should trust the developer to go and find the right link for themselves?

Reply:
Oh no sorry I parsed it wrong late in the day.